documentation/user_docs/docs/user_guide/Getting_started.rst

```diff
@@ -482,85 +655,201 @@ This should give a plot that looks like this:
+
+.. note::
+
+   As with ``func_eval``, the parameters passed to the function can
+   either take the form of a vector of numerical parameters, or a cell array
+   comprising any other form of input.
+
+
+Bose Temperature
+----------------
+
+In this example we will look at the generalised application function ``apply``.
+
+``apply`` is a somewhat advanced function which requires some knowledge of the
+underlying ``sqw`` object structure but allows us to apply arbitrary functions
+affecting more than just the signal to an object is a concise and flexible
+manner.
+
+For this example we will be applying a Bose population correction, first we define
+the function:
+
+.. math::
+
+   s' = \left(1 - \mathrm{e}^{ \frac{\sigma{}E}{T} }\right) s
+
+where :math:`s` is the signal intensity, :math:`T` is temperature, :math:`E` is is the energy, and :math:`\sigma = -11.6044`
+
+::
+
+   function pix = bose_population(pix, T)
+      pix.signal = pix.signal .* (1 - exp(-11.6044 .* pix.dE ./ T))
+   end
+
+Here we can see that this function takes ``pix`` as an argument which is the
+``PixelData`` object which contains all of the pixel information that underlies
+the ``sqw`` object. We are directly transforming the ``signal`` of the pixels with respect to ``E`` and the other input variable ``T`` (temperature)
+
+
+We can then simply apply the function to transform the underlying data of the ``sqw``.
+
+::
+
+   w_apply = w_template.apply(@bose_population, {50})
+
+This subsequently evaluates the correction with a temperature of ``50``.
+
+.. note::
+
+   Because the ``apply`` function is a somewhat low-level function, and it can take
+   multiple functions as arguments (as a cell-array of function handles) and
+   thus multiple sets of arguments, arguments are **always** passed as
+   cell-arrays or cell-arrays of cell-arrays in the case of multiple functions.
 
 
 Fitting
 =======
 
-You can also use Horace to fit your data. It can take quite a long time for the fit to converge, so it is therefore a
-good idea to provide a good initial guess of the fit parameters. You can work these out simulating and then comparing
-the result to the data by eye.
+You can also use Horace to fit parameters to your data. Since this a more
+complicated subject, for an introduction and overview of how to use the fitting
+functions, please read :ref:`Fitting data<manual/Multifit:Multifit>`.
+
+
+.. warning::
 
-For an introduction and overview of how to use the following fitting functions, please read
-:ref:`Fitting data<manual/Multifit:Multifit>`. For comprehensive help, please use the Matlab documentation for the various fitting
-functions that can be obtained by using the ``doc`` command, for example ``doc d1d/multifit`` (for fitting function like
-Gaussians to d1d objects) or ``doc sqw/multifit_sqw`` (fitting models for S(Q,w) to sqw objects).
+   For comprehensive help with anything in this guide, please use the Matlab
```

Comment:
do you mean anything in this guide? the next lines refer specifically to fitting